CHANGELOG.md

```diff
@@ -1,5 +1,271 @@
+- Added `debugName` parameter to `renderApplication` to use as the display name for the React root tree ([eeb36f4709](https://github.com/facebook/react-native/commit/eeb36f470929c2fdd8e1ed69898a5ba9144b8715) by [@rubennorte](https://github.com/rubennorte))
+- Event handlers return `mixed` type instead of `void`. ([7f005e6077](https://github.com/facebook/react-native/commit/7f005e6077398d2a7f31cb78313c90c667050792) by [@nadiia](https://github.com/nadiia))
+- Implement `sendAccessibilityEvent` in the React(Fabric/non-Fabric) renderer ([99b7052248](https://github.com/facebook/react-native/commit/99b7052248202cee172e0b80e7ee3dfb41316746) by [@JoshuaGross](https://github.com/JoshuaGross))
+- Added `displayName` to some RN contexts to make them more easy to differentiate when debugging. ([68a476103a](https://github.com/facebook/react-native/commit/68a476103a95be77f4fc7c582e52cc94946de1b4) by [@bvaughn](https://github.com/bvaughn))
+- Extend `react-native/babel-plugin-codegen` to generate TurboModule JS codegen ([b6a72bac69](https://github.com/facebook/react-native/commit/b6a72bac698486a96748365c8f67c8177afe317e) by [@RSNara](https://github.com/RSNara))
+- Adding support for `cancelOnBackground` for UserFlow ([0d4985900b](https://github.com/facebook/react-native/commit/0d4985900b52d5def22fce4371c2259ee65368ee) by [@dmitry-voronkevich](https://github.com/dmitry-voronkevich))
+- Introducing RuntimeScheduler module ([eb13baf2a6](https://github.com/facebook/react-native/commit/eb13baf2a687b53dde04b9a336f18629d94f4b79) by [@sammy-SC](https://github.com/sammy-SC))
+- Add ViewConfig for MultilineTextInput ([10f8c1a1c0](https://github.com/facebook/react-native/commit/10f8c1a1c0dd3301daab32201462a41be1086345) by [@p-sun](https://github.com/p-sun))
+- Roll out TurboModule Promise Async Dispatch ([5c4f145e33](https://github.com/facebook/react-native/commit/5c4f145e33d92969f8a86284360a5a2f09308500) by [@RSNara](https://github.com/RSNara))
+
+#### Android specific
+
+- Add `getRecommendedTimeoutMillis` to AccessibilityInfo ([d29a7e7a89](https://github.com/facebook/react-native/commit/d29a7e7a89f4e5e3489e9723979426bb1b6f0674) by [@grgr-dkrk](https://github.com/grgr-dkrk))
+- TalkBack now announces "unselected" when changing `accessibilityState.selected` to false. ([73bc96ecf9](https://github.com/facebook/react-native/commit/73bc96ecf9a16d420533c12e9e1812ffe21c10a2) by [@yungsters](https://github.com/yungsters))
+- Fbjni version bump to 0.0.3 ([24f9f75bf6](https://github.com/facebook/react-native/commit/24f9f75bf66b8f32a117ba9f9dea3c65b35b1e00) by [@IvanKobzarev](https://github.com/IvanKobzarev))
+- Added `Nullable` annotation ([903351f22e](https://github.com/facebook/react-native/commit/903351f22ee8e0bced19257ddee7fdf8a7516278))
+- Basic definition for react gradle plugin ([dbbc1c1624](https://github.com/facebook/react-native/commit/dbbc1c16244415aa6e5f40abadec352a42ad30ad) by [@ShikaSD](https://github.com/ShikaSD))
```

Comment:
This doesn't seem like it impacts the public release yet, it seems mostly for RNTester